test/lint/lint-tests.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+Check the test suite naming conventions
+"""
+
+import re
+import subprocess
+import sys
+
+
+def grep_boost_fixture_test_suite():
+    command = [
+        "git",
+        "grep",
+        "-E",
+        r"^BOOST_FIXTURE_TEST_SUITE\(",
+        "--",
+        "src/test/**.cpp",
+        "src/wallet/test/**.cpp",
+    ]
+    return subprocess.run(command, stdout=subprocess.PIPE, universal_newlines=True)
```

Comment:
Would prefer to use `check_output` here. This would make the script automatically fail if the git grep returns an error status. It also autmatically does `stdout=PIPE` and returns `stdout`.

Reply:
Done